game.py

```diff
@@ -299,12 +304,12 @@ def _test_game_over(self, new_dot):
         return False
 
     def _game_over(self):
-        best_time = 0
-        best_seconds = 0
-        best_minutes = 0
+        best_seconds = self.best_time % 60
+        best_minutes = self.best_time // 60
```

Comment:
This can be on one line too.

Reply:
these are different variables, not sure if I can merge it.

Reply:
Yeah didn't notice that, thanks.